### PR TITLE
qt5: update to 5.15.5

### PR DESCRIPTION
**Pull request by chrstphrchvz**
#### Description
https://github.com/qt/qtreleasenotes/blob/dev/qt/5.15.5/release-note.md

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [x] security fix

###### Tested on

macOS 12.4
Xcode 13.4 / Command Line Tools 13.4

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: aqua/qt5/Portfile [hunk 1]
```diff
@@ -1827,6 +1827,10 @@ foreach {module module_info} [array get modules] {
 
             # special case
             if { ${module} eq "qtmultimedia" } {
+                PortGroup           conflicts_build 1.0
+
+                # avfmediaplayerserviceplugin.h:55: Error: Undefined interface
+                conflicts_build-append ${subport}
```
Comment (chrstphrchvz) [inline aqua/qt5/Portfile:1833]: The error occurred during this command:

```
/opt/local/libexec/qt5/bin/moc -DQT_NO_LINKED_LIST -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_LINKED_LIST -DQT_DEPRECATED_WARNINGS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DGL_SILENCE_DEPRECATION -DQT_NO_EXCEPTIONS -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -DQT_NO_DEBUG -DQT_NO_KEYWORDS -DQT_PLUGIN -DQT_MULTIMEDIAWIDGETS_LIB -DQT_MULTIMEDIA_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/src/plugins/avfoundation/mediaplayer/.moc/moc_predefs.h -I/opt/local/libexec/qt5/mkspecs/macx-clang -I/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/src/plugins/avfoundation/mediaplayer -I/opt/local/libexec/qt5/include -I/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/include/QtMultimedia/5.15.4 -I/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/include/QtMultimedia/5.15.4/QtMultimedia -I/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/include/QtMultimediaWidgets/5.15.4 -I/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/include/QtMultimediaWidgets/5.15.4/QtMultimediaWidgets -I/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/include -I/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/include/QtMultimediaWidgets -I/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/include/QtMultimedia -I/opt/local/libexec/qt5/lib/QtWidgets.framework/Headers/5.15.4 -I/opt/local/libexec/qt5/lib/QtWidgets.framework/Headers/5.15.4/QtWidgets-I/opt/local/libexec/qt5/lib/QtGui.framework/Headers/5.15.4 -I/opt/local/libexec/qt5/lib/QtGui.framework/Headers/5.15.4/QtGui -I/opt/local/libexec/qt5/lib/QtWidgets.framework/Headers -I/opt/local/libexec/qt5/lib/QtGui.framework/Headers -I/opt/local/libexec/qt5/lib/QtCore.framework/Headers/5.15.4 -I/opt/local/libexec/qt5/lib/QtCore.framework/Headers/5.15.4/QtCore -I/opt/local/libexec/qt5/lib/QtNetwork.framework/Headers -I/opt/local/libexec/qt5/lib/QtCore.framework/Headers -I/opt/local/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk/usr/include/c++/v1 -I/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/13.1.6/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk/usr/include -I/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include -F/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_aqua_qt5/qt5-qtmultimedia/work/qtmultimedia-everywhere-src-5.15.4/lib -F/opt/local/libexec/qt5/lib avfmediaplayerserviceplugin.h -o .moc/moc_avfmediaplayerserviceplugin.cpp
avfmediaplayerserviceplugin.h:55: Error: Undefined interface
```

I did not encounter this issue during the 5.15.3 update; not sure whether change in 5.15.4 (see https://github.com/qt/qtmultimedia/compare/v5.15.3-lts-lgpl..v5.15.4-lts-lgpl) may be causing this, or if there is something like an include path precedence issue that has been around much longer.

**Comment by reneeotten** [comment]
> This is ready for review

You commented yourself about a build failure - has that been resolved or does the build still fail?

**Comment by chrstphrchvz** [comment]
> > This is ready for review
> 
> You commented yourself about a build failure - has that been resolved or does the build still fail?

My comment was meant just as a note pointing out that although I do not know what the exact cause for the build failure is, it has been resolved by adding conflicts_build.

**Comment by chrstphrchvz** [comment]
I will revise this to update to 5.15.5 instead

**Comment by chrstphrchvz** [comment]
Revised to include changes for CVE-2021-38593 backported from 5.15.6 (which will be publicly released in September).

**Review by mascguy — APPROVED** [review]

**Pull request merged**